roles/angular_cli/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# tasks file for angular_cli
+- name: Install required packages
+  apt:
+    name: [ 'npm' ]
```

Comment:
In this case, installing the default npm using the default package manager, is sufficent, or you don't need to build your applications against a specific version npm?